classes/blocks/class-carouselheader.php

```diff
@@ -127,15 +127,17 @@ public function prepare_data( $fields ): array {
 	 * @return string The block's content string.
 	 */
 	public function render_hydratable( $attributes, $content ) {
+		if ( ! empty( $content ) ) {
+			return $content;
+		}
 		if ( ! empty( $attributes['slides'] ) && empty( $attributes['slides'][0]['image_url'] ) ) {
 			$attributes['slides'] = self::get_slides_image_data( $attributes['slides'] );
 		}
 
 		$json = wp_json_encode( [ 'attributes' => $attributes ] );
 
 		// When we remove "beta" from the block name, we'll be able to use as_hydratable_block from class-base-block.php.
-		return '<div data-hydrate="planet4-blocks/carousel-header-beta" data-attributes="' . htmlspecialchars( $json ) . '">'
-		. trim( $content )
+		return '<div data-render="planet4-blocks/carousel-header-beta" data-attributes="' . htmlspecialchars( $json ) . '">'
```

Comment:
This might be good enough as a temporary solution. Rendering front end would temporarily have worse performance than the hydrated version, so we would want to migrate content after deploy. But it's probably still better performance than the old version.